MCprep_addon/mcprep_ui.py

```diff
@@ -95,9 +95,9 @@ def restart_layout(layout):
 	alert_row.alert = True
 	alert_row.operator(
 		"wm.quit_blender",
-		text="Restart blender",
+		text=env._("Restart blender"),
```

Comment:
Related to my point above, being confused about these.

Is this "Restart blender" used as a key lookup somewhere? (Are these env._() calls used to generate the po files... for then manual editing..?)

Reply:
`env._` is what does the actual mapping of a string to its corresponding translation, like how `translate_str` in the first attempt of i18n maps strings to translations. In addition, it's also used by `xgettext` to generate POT files (PO templates). 

Normally this would be `_` in the global scope, but to avoid unintentional breakage I've put it in `env`.

Reply:
Resolving this, knowing these calls are necessary to generate the template files.